### PR TITLE
Update FontAwesome from 4.7 and small fix

### PR DESCRIPTION
**Pull request by yamabiiko**
Views updated to the new names of the functions and icons of FontAwesome 5 to be able to use it with the gem "font-awesome-sass". Also fix for issue #47

### CODE REVIEW

File: app/views/main/mining.html.slim [hunk 1]
```diff
@@ -34,7 +35,8 @@ section#mining-effort
       .col-lg-6.col-md-6.text-center
         .service-box.mt-5.mx-auto
           = link_to "https://sia.tech/", target: "_blank", rel: "nofollow" do
-            = image_tag "siacoin", width: 192, class: 'sr-mining'
+            = image_tag "sc", width: 160, class: 'sr-mining'
+            p
```
Comment (FiloSpaTeam) [inline app/views/main/mining.html.slim]: Why a single p here? and above too :question: :dancer:

File: app/views/main/technicaladvice.html.slim [hunk 2]
```diff
@@ -32,7 +32,7 @@ article
             .col-4.col-sm-6.pr-0.text-right
               span class="fa-stack fa-3x"
                 i class="fa fa-circle fa-stack-2x"
-                i class="fa fa-commenting-o fa-stack-1x text-white"
+                i class="fa fa-comment-alt fa-stack-1x text-white"
```
Comment (FiloSpaTeam) [inline app/views/main/technicaladvice.html.slim]: You can substitute here with function too?

File: app/views/posts/edit.html.slim [hunk 3]
```diff
@@ -41,17 +40,17 @@ br
...
       - if @last_revision.nil?
         = link_to new_post_revision_path(post_id: @post.friendly_id), class: 'btn btn-primary btn-block rounded mb-2' do
-          = fa_icon('plus', text: t('.add_revision'))
+          = icon('fas', 'plus', t('.add_revision'))
       - else
         .row.form-group
           .col-4.text-center.collapsable data-col="col-4" data-collapsed="false" style="border-right: 3px solid #DC911B;"
             a.exclude.collapsable-toggle.mr-2.pull-right  href="javascript:;"
-              = fa_icon 'compress', class: 'fa-2x exclude'
+              = icon 'compress', class: 'fa-2x exclude'
```
Comment (FiloSpaTeam) [inline app/views/posts/edit.html.slim]: Add brackets or remove from all. My fault first of all :+1:

**Comment by yamabiiko** [comment]
Changes done, also done a fix for #47

**Pull request merged**